cdap-docs/build.sh

```diff
@@ -395,20 +246,20 @@ function build_docs_inner_level() {
     echo "--------------------------------------------------------"
     echo
     cd $SCRIPT_PATH/${i}
-    ./build.sh ${1} ${ARG_2}
+    ./build.sh ${1}
```

Comment:
It's not calling itself: it is calling the `build.sh` in each manual, each of which is customized for the manual. They all share the `common-build.sh`. 

The idea is that the same command can be used at both the individual manual level and at the doc-set level: `build.sh docs`. It makes the edit-build-check cycle simple.

The top level script changes into each manual, and runs the same command there, but it is running different code.